### DISCUSSION TITLE
Java/Quarkus for AI app development?

### DISCUSSION THREAD

**Discussion by aelashry777** [quarkusio/quarkus#44350] [Community]
Hello, 

Is anyone developing or has considered developing AI applications with Java? 

- If yes, are you using Quarkus or other frameworks? What challenges have you come across/what would you recommend? 
- If not, why not? Are you using other frameworks or languages for AI app development?  

Interested to see what people are exploring. Thank you.

**Comment by gsmet** [comment]
@aelashry777 could you clarify what you imply by AI? LLM or broader?

You probably know about it already but as for LLM, we have the [https://github.com/quarkiverse/quarkus-langchain4j](Quarkus LangChain4j project) that is very active.

**Reply by aelashry777** [reply]
interested in broad AI use cases, LLMs, ML, etc. Thank you, I will try that thread as well.
